aicsimageio/readers/reader.py

```diff
@@ -171,7 +173,7 @@ def current_scene_index(self) -> int:
         """
         return self.scenes.index(self.current_scene)
 
-    def set_scene(self, scene_id: str):
+    def set_scene(self, scene_id: str) -> None:
```

Comment:
Why is the return type None here?

Reply:
Because python functions by default return `None`.